src/main/java/com/school/project/model/types/UserAccountType.java

```diff
@@ -8,4 +8,15 @@
 
     UserAccountType(Integer id, String name) {
     }
+
+    public static UserAccountType getTypeById(Integer id) {
+        if (id==1){
+            return UserAccountType.TEACHER;
+        }
+        if (id==2){
+            return UserAccountType.ADMIN;
+        }
+        return UserAccountType.STUDENT;
```

Comment:
it doesn't work like this. what . will happen if tomorrow I will need 10 enums? think about more generic solution. there is a static function that returns all the enum of certain class.